### PR TITLE
Bykvist guest essay

### PR DESCRIPTION
**Pull request by whyboris**
😎

### CODE REVIEW

File: content/guest-essays/uncertainty-and-utilitarianism.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+year: "2022"
+date: 2022-08-08T10:10:55-04:00
+author: "Krister Bykvist"
+draft: false
+layout: "book"
+menu: "guest"
+weight: 8
+image: "/img/essays/criminal-law.jpg"
+imageCaption: "Uncertainty and Utilitarianism"
+summary: "summary"
```
Comment (whyboris) [inline content/guest-essays/uncertainty-and-utilitarianism.md:13]: @rychappell - please add one in sometime, or email me and I'll push the changes 🤝

File: content/guest-essays/uncertainty-and-utilitarianism.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+It is more controversial what you should do when the confidence and the value difference do not line up nicely in this way. Here we can follow the lead of what is reasonable to do under empirical uncertainty.[^7]
+
+Suppose Julia considers whether to speed around a blind corner. She thinks it is unlikely that anyone is crossing the road immediately around the corner, but she is not sure. If she speeds and hits someone, she will certainly severely injure them. If she goes slowly, she certainly will not injure anyone, but will get to work slightly later. Here the right option is clearly not to speed, even though it is unlikely that someone will cross the road. The situation has the following structure (the numbers are only illustrative):
+
+|               |  Someone will cross <br> (Probability: 0.2) |No one will cross <br> (Probability: 0.8) |
```
Comment (whyboris) [inline content/guest-essays/uncertainty-and-utilitarianism.md:85]: @rychappell I wonder if the table will look better splitting the line after `<br>` into a dedicated row:

|    | Someone will cross | No one will cross | 
|  --- | --- | --- |
| Probability | 0.2 | 0.8 |
| **Speed** | -20 | 15 | 
| **Not speed | 10 | 10 |

File: content/guest-essays/uncertainty-and-utilitarianism.md (hunk 1 above)
Comment (rychappell) [reply content/guest-essays/uncertainty-and-utilitarianism.md:13]: done!

File: content/guest-essays/uncertainty-and-utilitarianism.md (hunk 2 above)
Comment (rychappell) [reply content/guest-essays/uncertainty-and-utilitarianism.md:85]: Best to leave as is (i.e. probabilities included in the titles); too confusing to have some rows describing probabilities and some describing utilities.

**Pull request merged**